docs-devsite/vertexai.imagenmodelparams.md

```diff
@@ -20,12 +25,17 @@ export interface ImagenModelParams
 
 |  Property | Type | Description |
 |  --- | --- | --- |
-|  [generationConfig](./vertexai.imagenmodelparams.md#imagenmodelparamsgenerationconfig) | [ImagenGenerationConfig](./vertexai.imagengenerationconfig.md#imagengenerationconfig_interface) |  |
-|  [model](./vertexai.imagenmodelparams.md#imagenmodelparamsmodel) | string |  |
-|  [safetySettings](./vertexai.imagenmodelparams.md#imagenmodelparamssafetysettings) | [ImagenSafetySettings](./vertexai.imagensafetysettings.md#imagensafetysettings_interface) |  |
+|  [generationConfig](./vertexai.imagenmodelparams.md#imagenmodelparamsgenerationconfig) | [ImagenGenerationConfig](./vertexai.imagengenerationconfig.md#imagengenerationconfig_interface) | <b><i>(BETA)</i></b> Configuration options for generating images with Imagen. |
+|  [model](./vertexai.imagenmodelparams.md#imagenmodelparamsmodel) | string | <b><i>(BETA)</i></b> The Imagen model to use for generating images. For example: <code>imagen-3.0-generate-002</code>.<!-- -->Only Imagen 3 models (named <code>imagen-3.0-*</code>) are supported.<!-- -->See [model versions](https://cloud.google.com/vertex-ai/generative-ai/docs/image/model-versioning) for a full list of supported Imagen 3 models. |
```

Comment:
I'd like to link to our docs for this model versions (there are waaaaay too many versions listed in the Vertex AI docs).

The link will be https://firebase.google.com/docs/vertex-ai/models

Note that the link will 404 until launch day